python/ray/rllib/train.py

```diff
@@ -16,36 +17,55 @@
 
 parser = argparse.ArgumentParser(
     description=("Train a reinforcement learning agent."))
-parser.add_argument("--env", required=True, type=str)
-parser.add_argument("--alg", required=True, type=str)
-parser.add_argument("--config", default="{}", type=str)
-parser.add_argument("--upload-dir", default="file:///tmp/ray", type=str)
+parser.add_argument("--redis-address", default=None, type=str,
+                    help="The Redis address of the cluster.")
+parser.add_argument("--env", required=True, type=str,
+                    help="The gym environment to use.")
+parser.add_argument("--alg", required=True, type=str,
+                    help="The reinforcement learning algorithm to use.")
+parser.add_argument("--num-iterations", default=sys.maxsize, type=int,
+                    help="The number of training iterations to run.")
+parser.add_argument("--config", default="{}", type=str,
+                    help="The configuration options of the algorithm.")
+parser.add_argument("--upload-dir", default="file:///tmp/ray", type=str,
+                    help="Where the traces are stored.")
 
 
 if __name__ == "__main__":
     args = parser.parse_args()
+    json_config = json.loads(args.config)
 
-    ray.init()
+    if args.redis_address:
```

Comment:
We actually don't need this case. Since the default value of `args.redis_address` is `None`, we can just do `ray.init(redis_address=args.redis_address)`.